src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.api.ts

```diff
@@ -21,12 +21,16 @@ export const getManifestResource = (
         return getManifestResourceHelmApps(ad, podName, nodeType, isResourceBrowserView, selectedResource)
     }
     const cn = ad.resourceTree.nodes.filter((node) => node.name === podName && node.kind.toLowerCase() === nodeType)[0]
-
-    return get(
-        `api/v1/applications/${ad.appName}-${ad.environmentName}/resource?version=${cn.version}&namespace=${
-            ad.namespace
-        }&group=${cn.group || ''}&kind=${cn.kind}&resourceName=${cn.name}`,
-    )
+    selectedResource = {
```

Comment:
What is the difference in `selectedResource` coming from props and the one in line 24?

Reply:
The selectedResource prop for argocd returns undefined hence have to update the selectedResource data by filtering for current node details.